### DISCUSSION TITLE
Telafi

### DISCUSSION THREAD

**Discussion by Batur06** [IBAX-io/go-ibax#1842] [Bug Submission]
Görevler sırasında gerçekleşip yuklenmeyen puanların telafiisi  olacak mı bu konuda bı aciklama yapılacak mı sorun sizle mi alakalı yoksa twter le mi şeffaf olmalısınız

**Comment by emryldz86** [comment]
Twitter API sinin 2 milyon sınırı varmış ondan kaynaklanmış kanka apisi ne bilmiyorum ama günahı Twitterın boynuna 😀

**Comment by Skips1** [comment]
Ne durumdayız

**Comment by Abdullahoda** [comment]
Son 3 retweet puanlarını geri alıcaklar mış haksızlık olmasın diye mesajıma cevaplandı dersen sevinirim kardeşim
